lib/hsa/hc_profile.cpp

```diff
@@ -0,0 +1,29 @@
+#include "hc_profile.hpp"
+#include "CXLActivityLogger.h"
+
+cxlProfiler::cxlProfiler() {
+    if (getenv("HCC_CODEXL_PROFILING")) {
+        enabled = 1;
+        cxlMarker::enabled = 1;
+        amdtInitializeActivityLogger();
```

Comment:
@otauqir static dependency to CodeXL is undesirable. It means every HCC application now would be linked with CodeXL library. Please consider improve the design to use dynamic libraries.